c++/src/capnp/rpc-twoparty.h

```diff
@@ -84,9 +91,14 @@ class TwoPartyVatNetwork: public TwoPartyVatNetworkBase,
   // to be sent on this connection. This may be useful for backpressure.
 
   size_t getCurrentQueueCount() { return currentQueueCount; }
-  // Get the count outgoing messages that are currently queued in memory waiting
+  // Get the count of outgoing messages that are currently queued in memory waiting
   // to be sent on this connection. This may be useful for backpressure.
 
+  kj::Duration getOldestQueuedMessageAge();
```

Comment:
The naming here is a bit questionable, since it returns the age of the most recently *sent* message, not the oldest queued message. Or even if the name is ok, the comment could clarify what this value actually represents more accurately than by describing it as the "oldest outgoing message that is currently queued in memory waiting to be sent on this connection".

And doing the queue count check within here rather than always returning the age of the most recently sent message might theoretically be removing useful info -- maybe someone cares how recently a message was sent on the connection and this could be a good way to get that -- but at the same time there hasn't been any clear need for that functionality yet, so I suppose we don't need to worry about it.

Reply:
Yeah I'd argue the naming is reasonable, because if the next message in the queue _hasn't_ been sent yet, then we are in fact still in the process of sending the one before it, e.g. the previous message is only half-written to the socket. So in some sense it's still "queued"...